### PR TITLE
updates for example of using a VF

### PR DESCRIPTION
**Pull request by lamek**

### CODE REVIEW

**Comment by lamek** [comment]
WIP - I'm going to modularize this content.

**Comment by lamek** [comment]
This has been modularized and is ready for review.

File: modules/nw-sriov-example-vf-function-in-pod.adoc [hunk 1]
```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple-networks/configuring-sr-iov.adoc
+
+[id="example-vf-use-in-pod_{context}"]
+= Example use of a virtual function (VF) in a Pod
+
+You can run a remote direct memory access (RDMA) or a Data Plane Development Kit (DPDK) application in a Pod with SR-IOV VF attached.
+In the following example, a Pod is using a VF in RDMA mode:
```
Comment (mburke5678) [inline modules/nw-sriov-example-vf-function-in-pod.adoc]: Should this be +This example shows a pod using a VF in RDMA mode:_
The current construct is a but awkward and changing would make it parallel the next example.

File: modules/nw-sriov-example-vf-function-in-pod.adoc [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  - name: hugepage
+    emptyDir:
+      medium: HugePages
+----
+
+An optional library is available to aid the application running in a container in gathering network information associated with a pod. This library is called 'app-netutil'. See the library's source code in the link:https://github.com/openshift/app-netutil[`app-netutil` GitHub repo]. 
+
+This library is intended to ease the integration of the SR-IOV VFs in DPDK mode into the container. The library provides both a GO API and a C API, as well as examples of using both languages.
+
+There is also a sample docker image, 'dpdk-app-centos', which can run one of the following DPDK sample applications based on an environmental variable in the pod-spec: l2fwd, l3wd or testpmd. This docker image provides an example of integrating the 'app-netutil' into the container image itself. The library can also integrate into an init-container which collects the desired data and passes the data to an existing DPDK workload.
```
Comment (mburke5678) [inline modules/nw-sriov-example-vf-function-in-pod.adoc]: s/docker/Docker. Capitalized Docker seems more prevalent in our docs.

**Comment by mburke5678** [comment]
@lamek A couple of nits. Otherwise LGTM

File: modules/nw-sriov-example-vf-function-in-pod.adoc (hunk 1 above)
Comment (lamek) [reply modules/nw-sriov-example-vf-function-in-pod.adoc]: That's much better. Great suggestion.

**Pull request merged**